### PR TITLE
[#582] Updated version of xtext-gradle-plugin

### PR DESCRIPTION
**Pull request by FStolte**
Related to issue [eclipse/xtext-core#582]
- updated from 1.0.19 to 1.0.20
- updated the version in the gradle file
- updated the version used by the wizard
- updated the version in one example

Signed-off-by: Florian Stolte <fstolte@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please adjust all the tests as well

**Comment by FStolte** [comment]
For some reason I did not find the other tests using the old xtext-gradle-plugin version yesterday. I will adjust them now.

**Comment by FStolte** [comment]
Ok, now all tests should be covered.

File: gradle/versions.gradle [hunk 1]
```diff
@@ -1,29 +1,29 @@
-/*
- * Base project version and versions of common dependencies.
- */
-
-version = '2.14.0-SNAPSHOT'
-
```
Comment (cdietrich) [inline gradle/versions.gradle]: am not sure which kind of line endings we want to have here @szarnekow @kthoms any idea

**Pull request merged**